engine/src/test/java/org/hibernate/validator/test/constraints/annotations/hv/age/AgeValidatorConstrainedTest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.test.constraints.annotations.hv.age;
+
+import static org.hibernate.validator.testutil.ConstraintViolationAssert.assertNoViolations;
+import static org.hibernate.validator.testutil.ConstraintViolationAssert.assertThat;
+import static org.hibernate.validator.testutil.ConstraintViolationAssert.violationOf;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
```

Comment:
seems to be unused import - http://ci.hibernate.org/job/hibernate-validator-PR/1062/console

Reply:
Done